src/View/ViewVarsTrait.php

```diff
@@ -26,6 +26,22 @@
 {
 
     /**
+     * The name of default View class.
+     *
+     * @var string
+     */
+    public $viewClass = null;
+
+    /**
+     * View instance.
+     *
+     * Won't be set until after ViewVarsTrait::createView() is called.
+     *
+     * @var \Cake\View\View
+     */
+    public $_view;
```

Comment:
I would like to deprecate `Controller::$layout`,  `Controller::$view`,  `Controller::$theme` too and promote directly using the View class' property as shown above.